tests/roles/backend_services/tasks/main.yaml

```diff
@@ -94,25 +113,20 @@
         template:
           external-ids: {}
 
-      neutron:
-        enabled: false
-        template: {}
-
-      nova:
+      placement:
         enabled: false
         template: {}
 
-      ironic:
-        enabled: false
-        template:
-          ironicConductors: []
+      rabbitmq:
```

Comment:
Now i see i forgot the net isolation parameters in the test suite here for backend services. The one in Keystone test seems to have worked tho.